runtime/appruntime/metrics/json_metrics_exporter.go

```diff
@@ -0,0 +1,18 @@
+package metrics
+
+import (
+	"encore.dev/appruntime/config"
+	"encore.dev/appruntime/metrics/json_based"
+)
+
+func init() {
+	registerProvider(providerDesc{
```

Comment:
See earlier comment: I think we shouldn't need the JSON metrics to be considered a metrics exporter